apps/dolly-frontend/src/main/js/src/components/fagsystem/skatteetaten/visning/SkatteetatenVisning.tsx

```diff
@@ -26,18 +27,26 @@ export const SkatteetatenVisning = ({ data, loading }: SkatteetatenVisningProps)
 	}
 
 	const tjenestepensjonavtaleListe = tenorRelasjoner.tjenestepensjonavtale
+	const harTjenestepensjonavtale = tjenestepensjonavtaleListe?.length > 0
+
 	const harDagligLederRolle = _get(tenorRelasjoner, 'brreg-er-fr')?.length > 0
```

Comment:
Pt er dette korrekt, men hvis Skatt åpner for flere roller er det bedre å si `harRoller` som også da vil fortsette å være riktig. 😃